lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -506,6 +508,23 @@ spec = do
                             || not (rangeHasUpperBound slotRange)
                             || upperBoundSucc slotRange == slotRange)
 
+        let f sps r = fmap (slotStartTime sps) <$> slotRangeFromTimeRange sps r
+        let sharedDescription = " (where f = fmap slotStartTime . \
+            \slotRangeFromTimeRange)"
+        it ("f timeRange ⊆ timeRange" ++ sharedDescription) $
```

Comment:
Hi @Anviking. This is a good test, and we should definitely have it. Though I'm fairly certain it duplicates "Rule 1" in the test above:

https://github.com/input-output-hk/cardano-wallet/pull/652/files#diff-ea01d42dc8106d3d2e18f8ec129c9fa3R494-R496

What do you think?

Reply:
I've checked, and I'm pretty sure that this property duplicates rule 1 from the existing property. So I've removed this property for now.